HW1/P5/P5.txt

```diff
@@ -1 +1,160 @@
-# Your discussion here
+Note that helper functions are stored in the "network_commands" python module in the directory above this one.
+I wanted to place all of my network code in the same place.
+
+###### Shortest path #######
+
+The shortest path between 'Harvard_University' and 'Kevin_Bacon' is of length 3. One example is
```

Comment:
The shortest path from Harvard University to Kevin Bacon has length 3, but the shortest path from Kevin Bacon to Harvard University has length 2...